.github/ISSUE_TEMPLATE/-issue-number---problem-description-.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "[Issue number - problem description]"
+about: This template standardizes issue reporting within the project.
+title: "[Issue]"
+labels: bug, documentation, duplicate, enhancement, good first issue, help wanted,
+  invalid, question, wontfix
```

Comment:
So if I open an issue, all these labels are added? I think we should rethink this. The labels will be added based on the checkbox in the `Extension` section. And for that, we will have to create a bot. 😅 

Unless there's an option for users to pick the area once open an issue here. For example, the flow can be something like: Open an Issue, Pick from `server`, `client`, or `general`. So then GH can assign the correct labels.